### DISCUSSION TITLE
[Atualização] Projeto open-source para ajudar pessoas com depressão (Birrépi)

### DISCUSSION THREAD

**Discussion by hectorgrecco** [frontendbr/forum#1146] [Dicas]
Fala, galera! Primeiramente gostaria de agradecer à todos que mandaram mensagens apoiando o projeto, o @LFeh por ajudar na divulgação e principalmente a galera que tá no grupo do Slack ([Clique aqui para entrar](http://bit.ly/SlackBirrepi)). Vou passar aqui o que já temos definido até agora:

**Nome:** Birrépi (Simplesmente uma versão abrasileirada da frase "Be Happy" (seja feliz em inglês), que define bastante o objetivo do projeto, que é ajudar pessoas com depressão a não totalmente "curar" isso, mas sim dar uma certa luz)

**Backlog:** Plataforma (ou comunidade, como preferirem) 100% open-source com o seguinte propósito:

1.  A pessoa com depressão ou qualquer outro tipo de "problema" envia uma mensagem na plataforma (podendo ser anonimamente ou não);
2. Qualquer usuário da plataforma responde a mensagem da pessoa com frases de apoio, músicas ou qualquer outra coisa que faça a outra pessoa se sentir bem. (Pra evitar haters/trolls, a ideia é que todas essas respostas sejam moderadas por estudantes de psicologia, Psicólogos formados, psiquiatras e cia. Além disso, será pedido o CPF da pessoa no registro pra diminuir a incidência desses trolls);
3. Um blog a parte ou dentro da plataforma com dicas de saúde mental, dicas para os familiares, amigos da pessoa com depressão, entre outros.
4. Um micro-serviço de buscar consultórios de Psicologia, Serviço Social, Psiquiatria, etc pela geolocalização do usuário. (Como o projeto é sem fins lucrativos, a ideia é retornar somente consultórios com atendimento gratuito. A API a ser utilizada é [essa aqui](http://bit.ly/APIEstabelecimentosdeSaude)).

**Tecnologias (Stack):**  Depois de vários debates e opiniões no grupo do Slack, ficou-se definido que serão utilizadas as seguintes tecnologias:

- **[Vue.js](https://vuejs.org)**, _para o Front-End_
- **Node.js** com o framework **[AdonisJs](https://adonisjs.com/)**, _para o Back-End_
- **PostgreSQL**, _para o banco de dados_

Essas tecnologias foram escolhidas por terem uma curva de aprendizado baixo e serem bem atuais por aí! No fim das contas, a aplicação será uma Single-Page Application (SPA) + PWA (Progressive Web App)

Com os aspectos iniciais do projeto já tomados, o próximo passo é modelar o BD, Back-end, Front-end e apresentar um MVP. Além disso, precisamos de um designer para desenvolver uma identidade visual para o projeto. Como o projeto é sem-fins lucrativos e 100% open-source, obviamente não temos recursos para pagar profissionais, todos os envolvidos estão determinados a concluir isso aqui pela "simples" vontade de ajudar!

**_[GRUPO DO SLACK](http://bit.ly/SlackBirrepi)_**
[**_REPOSITÓRIO OFICIAL_**](https://github.com/birrepi/birrepi)

Críticas, sugestões, indicações são sempre bem-vindas! Mais uma vez, meu muito obrigado à galera da Front-End Brasil pela força!!!! =)

**Comment by WandersonAlves** [comment]
Cês já possuem guidelines para contribuir? Topo muito entrar nessa

**Comment by hectorgrecco** [comment]
@WandersonAlves então... a priori precisamos modelar o BD do zero, e pelo visto será em Postgres! As tarefas serão organizadas no Trello e a comunicação entre as equipes de front-end e back-end será via canais separados do Slack. Deu pra entender mais ou menos? hehe

**Comment by WandersonAlves** [comment]
Deu sim kkkkk, em breve vou aparecer no slack quando ficar um pouco mais livre aqui

**Comment by kvnol** [comment]
Curti muito a ideia, seria massa criar uma organização geral e lá dentro ter repositórios para o back-end, front e design.

Aliás, fiz um esboço de como poderia ser o logo: https://codepen.io/kvnol/pen/LBYjPw

**Comment by DevCarlosAugusto** [comment]
To dentro, estou entrando no grupo do Slack.
